visma/simplify/simplify.py

```diff
@@ -165,6 +167,97 @@ def simplifyEquation(lToks, rToks):
+                            scope.append(i - 2)
+                            tokens1[i - 2].tokens, _, _, _, _ = expressionSimplification(tokens_now, scope, tokens1[i - 2].tokens)
+                        a = tokens1[i - 2]
+                        b = tokens1[i]
+                        c = a * b
+                        mulFlag = True
+                        expresstionMultiplication = True
+                        if isinstance(c, Expression):
+                            scope.append(i)
+                            c.tokens, _, _, _, _ = expressionSimplification(tokens_now, scope, c.tokens)
+                        tokens1[i] = c
+                        del tokens1[i - 1]
+                        del tokens1[i - 2]
+                        break
+        if not mulFlag:
+            break
+    if expresstionMultiplication:
+        animation.append(tokens1)
+        comments.append(['Multiplying expressions'])
+    expressionPresent = False
+    for i, _ in enumerate(tokens1):
+        if isinstance(tokens1[i], Expression):
+            expressionPresent = True
+            scope.append(i)
+            newToks, _, _, _, _ = expressionSimplification(tokens_now, scope, tokens1[i].tokens)
+            if not simToks:
+                simToks.extend(newToks)
+            elif (simToks[len(simToks) - 1].value == '+'):
+                if isinstance(newToks[0], Constant):
+                    if (newToks[0].value < 0):
+                        simToks.pop()
+                simToks.extend(newToks)
+            elif (simToks[len(simToks) - 1].value == '-'):
+                for _, x in enumerate(newToks):
+                    if x.value == '+':
+                        x.value = '-'
+                    elif x.value == '-':
+                        x.value = '+'
+                if (isinstance(newToks[0], Constant)):
+                    if (newToks[0].value < 0):
+                        simToks[-1].value = '+'
+                        newToks[0].value = abs(newToks[0].value)
+                elif (isinstance(newToks[0], Variable)):
+                    if (newToks[0].coefficient < 0):
+                        simToks[-1].value = '+'
+                        newToks[0].coefficient = abs(newToks[0].coefficient)
+                simToks.extend(newToks)
+        else:
+            simToks.extend([tokens1[i]])
+    simToks = tokenizer(tokensToString(simToks))
+    if expressionPresent:
+        animation += [simToks]
+        comments += [['Opening up all the brackets']]
+    if scope == []:
+        simToks, availableOperations, token_string, animExtra, commentExtra = simplifyNormal(simToks)
+        animExtra.pop(0)
+        animation += animExtra
+        comments += commentExtra
+    else:
+        availableOperations = ''
+        token_string = ''
+    if scope != []:
+        scope.pop()
+    return simToks, availableOperations, token_string, animation, comments
+
+
+def simplifyNormal(tokens):
```

Comment:
Rename `simplifyNormal` to `simplification`.
```suggestion
def simplification(tokens):
```